rdm/render.py

```diff
@@ -34,6 +34,34 @@ def join_to(foreign_keys, table, primary_key='id'):
     return joined
 
 
+def md_indent(snippet, header_shift=0):
+    lines = snippet.split('\n')
+    i = 0
+    in_code = False
+    if header_shift < 0:
+        while i > header_shift:
```

Comment:
I suspect there's a way to do this without using this while loop. See if you can find a way to simplify it out.